src/utils/image_server.cljs

```diff
@@ -26,7 +26,7 @@
     4))
 
 (defn get-account-image-uri
-  [port public-key image-name key-uid theme]
+  [{:keys [port public-key image-name key-uid theme ring?]}]
```

Comment:
the `ring?` here will be used later to disable ident ring for user with ens name